UnitedIncome/components
# Atoms/Loader -> TypeScript

### Description
> Please provide a brief description of your changes below.

Converts the `Loader` component to use TypeScript and react-testing-library.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Should be the same. Code changes only.
